app/views/shop/products/_shop_variant_no_group_buy.html.haml

```diff
@@ -9,7 +9,7 @@
       -# U+FF0D Fullwidth Hyphen-Minus
       －
     %input.variant-quantity{ type: "number", min: "0", max: "{{ available() }}",
-      ng: {model: "variant.line_item.quantity", max: "Infinity"}}>
+      ng: {model: "variant.line_item.quantity", attr: { id: "variant_line_item_quantity_{{variant.line_item.id}}" }, change: "updateCart(variant.line_item.quantity)", max: "Infinity"}}>
```

Comment:
It's much easier to include the id in the element declaration. More performant as Ruby code, too:

```haml
    %input.variant-quantity{ id: "variant_line_item_quantity_#{variant.line_item.id}", type: "number", min: "0", max: "{{ available() }}",
```

The id doesn't change. And we can probably simplify it:
```haml
    %input.variant-quantity{ id: "variant_#{variant.id}_quantity", type: "number", min: "0", max: "{{ available() }}",
```